### PR TITLE
Force truffle to estimate gas

### LINKED ISSUE #1524: Use estimated gas for CLI
Right now, the CLI uses a very high gas parameter that requires users to have a high ETH balance to send transactions. We should use truffle's gas estimation instead.

### PR DESCRIPTION
**Pull request by mrice32**
Fixes #1524.

### CODE REVIEW

**Comment by mrice32** [comment]
> Does this migrate contracts without issue?

Apparently not :/

**Comment by mrice32** [comment]
> > Does this migrate contracts without issue?
> 
> Apparently not :/

@nicholaspai looks to be working now. PTAL.

File: common/globalTruffleConfig.js [hunk 1]
```diff
@@ -21,7 +21,6 @@ const infuraApiKey = process.env.INFURA_API_KEY ? process.env.INFURA_API_KEY : "
 
 // Default options
 const gasPx = 20000000000; // 20 gwei
-const gas = 9000000; // Conservative estimate of the block gas limit.
```
Comment (chrismaree) [inline common/globalTruffleConfig.js:24]: could you not set `gas = undefined` here then only need to change this one line?

Comment (mrice32) [reply common/globalTruffleConfig.js:24]: Good point. Will update.

Comment (mrice32) [reply common/globalTruffleConfig.js:24]: Done.

**Pull request merged**